frontend/src/metabase/dashboard/dashboard.integ.spec.js

```diff
@@ -0,0 +1,73 @@
+import { PublicApi } from "metabase/services";
+import { fetchDashboard } from "./dashboard"
+
+import {
+    createTestStore,
+    login
+} from "metabase/__support__/integrated_tests";
+
+import { getParameterFieldValues } from "metabase/selectors/metadata";
+import { ADD_PARAM_VALUES } from "metabase/redux/metadata";
+
+// TODO Atte Keinänen 7/17/17: When we have a nice way to create dashboards in tests, this could use a real saved dashboard
+// instead of mocking the API endpoint
```

Comment:
We could probably just POST things to an API endpoint and create the dashboard in integration tests.